docs/hardware/devices/nano/index.mdx

```diff
@@ -36,7 +36,7 @@ The Nano G2 Ultra and Nano G1 Explorer have exactly the same Lora front-end circ
 - **LoRa Transceiver:**
   - Semtech SX1262
 - **Frequency Options:**
-  - _US (902.0Mhz to 928.0Mhz), EU_868 (869.4Mhz to 869.65Mhz), JP (920.8Mhz to 927.8Mhz), ANZ (915.0Mhz to 928.0Mhz), RU (868.7Mhz to 869.2Mhz), KR (920.0Mhz to 923.0Mhz), TW (920.0Mhz to 925.0Mhz), IN (865.0Mhz to 867.0Mhz), NZ_865 (864.0Mhz to 868.0Mhz), TH (920.0Mhz to 925.0Mhz), UA_868 (868.0Mhz to 868.6Mhz)._
+  - _US (902.0Mhz to 928.0Mhz), EU_868 (869.4Mhz to 869.65Mhz), JP (920.8Mhz to 927.8Mhz), ANZ (915.0Mhz to 928.0Mhz), RU (868.7Mhz to 869.2Mhz), KR (920.0Mhz to 923.0Mhz), TW (920.0Mhz to 925.0Mhz), IN (865.0Mhz to 867.0Mhz), NZ_865 (864.0Mhz to 868.0Mhz), TH (920.0Mhz to 925.0Mhz), UA_868 (868.0Mhz to 868.6Mhz), MY_919 (919.0Mhz to 924.0Mhz), SG_923 (917.0Mhz to 925.0Mhz)._
```

Comment:
What is `MY`? Was that change intentional?

Reply:
Malaysia was added a bit ago but I missed this reference so picked it off while I was in the neighborhood.